lyricsgenius/genius.py

```diff
@@ -395,25 +397,26 @@ def search_song(self, title=None, artist="", song_id=None,
                 print('Searching for "{s}"...'.format(s=title))
 
         if song_id:
-            result = self.song(song_id)['song']
+            search_term = "id:{id}".format(id=song_id).strip()
```

Comment:
Same thing about the `search_term` in `search_album`. If this was meant for the print statement down there, it's still not going to be used, because a wrong song ID will definitely result in an error.